apps/metabase/app.yaml

```diff
@@ -0,0 +1,83 @@
+{
+    "id": "/metabase",
+    "cmd": null,
+    "cpus": 2,
+    "mem": 2048,
+    "disk": 0,
+    "gpus": 0,
+    "instances": 1,
```

Comment:
why only 1? this will result in downtime during deploys

Reply:
Metabase is written to have only 1 instance running behind an LB according to an issue filed in the main metabase repo.

Reply:
wtf

Reply:
if this is the case, we'll need to think carefully about depending on this inside ocfweb

Reply:
I think it will be fine if we use metabase to generate the graphs and then cache everything using ocfweb's caching layer like we already do.